modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,35 +1,31 @@
 = What’s New in Studio
 
-Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
+Anypoint Studio 7.14.0 introduces new features to enhance the users' experience making the daily use of Studio easier, faster, and more agile. 
 
-This version also improves the general user experience including new features such as the Exchange Comprehensive Publication.
+== Improved Copy-Paste Experience
 
-== Explore and Debug Shared Flows
+Now, users have the possibility to select if a suffix is added to the copy-pasted Mule Palette elements and to the flows to avoid having similar long names.
 
-Studio now supports message flow visualization of a Mule configuration file loaded from a dependency, which enables exploring and debugging shared flows.
+== Deploy to CloudHub 2.0
 
-image::explore-and-debug-shared-flows.png["A Mule Configuration file inside a dependency is highlighted in the Package Explorer and its flow displayed on the main panel."]
+Aligned with the latest release, Studio allows developers to deploy their applications to https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub 2.0], which offers a wide variety of shared services.
 
-== Easy Management of DataWeave Libraries
+== Project Creation Made Easier
 
-Users now have easier access to manage DataWeave libraries through the Studio UI.
+The on-screen information has been simplified and now users can choose the view of API implementation when creating a project from Studio preferences:
 
-== New Exchange Comprehensive Publication Features
+* The API implementation section is now collapsable.
+* The API implementation tab layout is modified to get easier access to the most common options.
 
-The Exchange Comprehensive Publication features enhance the user experience in Studio when publishing assets to Exchange.
+image::api-implementation-collapsed.png["*API Implementation* section collapsed in the *New Mule Project* window."]
 
-* Custom fields are now available to add from the wizard.
-* Users can now create and edit Markdown files and deploy them along with the asset.
-* An icon can be added to new assets when publishing them to Exchange.
+== Increased Visibility and Readability
 
-== Improved General User Experience
+In line up with accessibility initiatives, the color contrast in Studio is enhanced to improve its visibility and readability.
```

Comment:
_In line up with_ accessibility initiatives, the color contrast _in Studio is enhanced_ to improve its visibility and readability. >
**Following** accessibility initiatives, **Studio has an enhanced** color contrast to improve visibility and readability.

Reply:
Thanks!